apps/condo/domains/news/schema/NewsItemRecipientsExportTask.test.js

```diff
@@ -113,12 +113,14 @@ describe('NewsItemRecipientsExportTask', () => {
             test('admin can', async () => {
                 const [objCreated] = await createTestNewsItemRecipientsExportTask(adminClient, adminClient.user, dummyO10n)
 
-                const [obj, attrs] = await updateTestNewsItemRecipientsExportTask(adminClient, objCreated.id)
+                await waitFor(async () => {
```

Comment:
Why to you need `waitFor` you can update item and inspect it right after.

Or you're just starting updating it infinitely until `v` is `3`. Seems like a hack, not a fix

Reply:
problem is that we can't guarantee if the worker will work in time. If it does, it will be v 3, if it doesn't, it will be v 2. I have looked at how it is implemented in other export tasks and it seems that it is common to do it via waitFor

if you know how you can ensure that check happens faster than the worker works and it is somehow the best solution just write about it

Reply:
done